### ISSUE TITLE
Unable to create context store: %userprofile% is not defined

### ISSUE THREAD

**Issue by kdhunter** [docker/for-win#8545]
### Expected behavior

Things that worked with 2.3.0.4 should still work with 2.3.0.5.  It's a fourth digit version bump, guys.

### Actual behavior

After updating to 2.3.0.5, we get "Unable to create context store: %userprofile% is not defined" errors, in situations in which 2.3.0.4 works perfectly.

### Information

We launch "docker build" commands from Python scripts.  After updating to 2.3.0.5, the scripts fail with the above error.

The work-around seems to be to disable the new "Enable Cloud Experience" feature in settings.

A similar report has been made over in the Mac area - https://github.com/docker/for-mac/issues/4919

Basically it appears 2.3.0.5 with the Cloud Experience enabled introduced some kind of dependency on environment variables that were not present on 2.3.0.4.

**Comment by stephen-turner** [comment]
Thanks for the report, @kdhunter. Obviously this was not intentional in a patch release. But you can revert to the old CLI in the settings, hopefully that will fix it.
